### ISSUE TITLE
Display message when no error was found

### ISSUE THREAD

**Issue by yoanmalie** [juanfran/gulp-scss-lint#74]
Hello,
i want to log a message when no error was found in all file. But this not really work because the message is repeated for each file. Like:
![lintcss](https://cloud.githubusercontent.com/assets/3357171/18670776/bc260038-7f41-11e6-90cb-d43ee22feb6a.png)

I use the custom error report:
```
ScssLintReporter = function(file) {
    if (!file.scsslint.success) {
        // list errors
    }
    else {
        util.log('Well done :) !')
    }
}
```
Is there a way to do this ? Or please, can you add the possibility to do it.

Thank

**Comment by juanfran** [comment]
Hi! Could you try this?


```js
gulp.task('a2', function() {
  var scssLintSuccess = true;
  var ScssLintReporter = function(file) {
     if (!file.scsslint.success) {
        // list errors
        scssLintSuccess = false;
     }   
   };

   return gulp.src('**/*.scss')
       .pipe(scsslint({customReport: ScssLintReporter})
       .on('end', function(x) {
            if (scssLintSuccess) {
                    util.log('Well done :) !');
             }
       })
});
```

**Comment by yoanmalie** [comment]
@juanfran Great, thanks !
